pfmask-to-pfsol compilation error

Hello All,

Though I have been successfully able to compile ParFlow up to v3.3.1, compilation is failing for me for version 3.4.0. I am on LSU's SMIC. I have attached my compilation script (mkparflow.sh), as well as the error log (results.txt) with the relevant portions showing what is throwing errors. Using the same compile script, if I change to release 3.3.1, compilation is successful and all tests run correctly. 

Is anybody else having issues upgrading? Any suggestions?

Thanks in advance!

Jack Cadigan, E.I.
Graduate Research Assistant
Department of Civil and Environmental Engineering
Louisiana State University
jcadig1@gmail.com

[ 94%] Building CXX object pftools/CMakeFiles/pfmask-to-pfsol.dir/pfmask-to-pfsol.cpp.o
/home/jcadig1/parflow4/parflow/pftools/pfmask-to-pfsol.cpp(126): error: qualified name is not allowed
  typedef std::numeric_limits< double > dbl;
          ^

/home/jcadig1/parflow4/parflow/pftools/pfmask-to-pfsol.cpp(126): error: expected a ";"
  typedef std::numeric_limits< double > dbl;
                             ^

/home/jcadig1/parflow4/parflow/pftools/pfmask-to-pfsol.cpp(136): error: name followed by "::" must be a class or namespace name
    pfsolFile.precision(dbl::max_digits10);
                        ^

/home/jcadig1/parflow4/parflow/pftools/pfmask-to-pfsol.cpp(201): error: name followed by "::" must be a class or namespace name
    vtkFile.precision(dbl::max_digits10);
                      ^

compilation aborted for /home/jcadig1/parflow4/parflow/pftools/pfmask-to-pfsol.cpp (code 2)
make[2]: *** [pftools/CMakeFiles/pfmask-to-pfsol.dir/pfmask-to-pfsol.cpp.o] Error 2
make[1]: *** [pftools/CMakeFiles/pfmask-to-pfsol.dir/all] Error 2
make: *** [all] Error 2

--- a/pftools/pfmask-to-pfsol.cpp
+++ b/pftools/pfmask-to-pfsol.cpp
@@ -38,6 +38,7 @@
 #include <fstream>
 #include <cassert>
 #include <cmath>
+#include <limits>
 
 /*
  * The mask directions.

